readme.md

```diff
@@ -112,6 +112,41 @@ The function throws an error if anything goes wrong, which can be handled by cat
 await mdToPdf({ content: '# Hello, World' }, { dest: 'path/to/output.pdf' });
 ```
 
+#### Table of content
+
+Place the following comment `<!-- TOC -->` where you want the table of content be generated from document headings
+
+Options for table of content are : 
+
+* `TOC_headings`: Set the table of content title ( default: 'Table of content' )
+* `TOC_depth`: Skip n level for the table of content generation ( default: 0 ) 
+ 
+If you want to skip the first level headings of your document, set level to 1. 
```

Comment:
This is a bit confusing. I would think that a TOC depth of 1 will only include level 1 headings (i. e. `h1`), and a depth of 3 would include `h1`, `h2` and `h3` headings.